src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1508,6 +1524,11 @@ void TorrentHandle::handleSaveResumeDataAlert(libtorrent::save_resume_data_alert
         resumeData["qBt-magnetUri"] = toMagnetUri().toStdString();
         resumeData["qBt-paused"] = isPaused();
         resumeData["qBt-forced"] = isForced();
+        // Both firstLastPiecePriority and sequential need to be stored in the
+        // resume data if there is no metadata, otherwise they won't be
+        // restored if qBittorrent quites before the metadata is retrieved:
```

Comment:
@sledgehammer999, @briankendall:  Actually, `is retrieved` is correct ("metadata" is treated as singular in most cases of English syntax).  

I agree with you about `quite`/`quit`, however.